e2e/tests/plugins/notebook/tags.e2e.spec.js

```diff
@@ -137,10 +146,55 @@ test.describe('Tagging in Notebooks', () => {
         await expect(page.locator('[aria-label="Search Result"]')).not.toContainText("Driving");
     });
     test('Tags persist across reload', async ({ page }) => {
-        await createNotebookEntryAndTags(page);
+        //Go to baseURL
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        // Create a clock object we can navigate to
+        await page.click('button:has-text("Create")');
+
+        // Click Clock
+        await page.click('text=Clock');
+        // Click button:has-text("OK")
+        await Promise.all([
+            page.waitForNavigation(),
+            page.locator('[name="mctForm"] >> text=My Items').click(),
+            page.locator('button:has-text("OK")').click()
+        ]);
+
+        await page.click('.c-disclosure-triangle');
+
+        const ITERATIONS = 4;
+        await createNotebookEntryAndTags(page, ITERATIONS);
+
+        for (let iteration = 0; iteration < ITERATIONS; iteration++) {
+            const entryLocator = `[aria-label="Notebook Entry"] >> nth = ${iteration}`;
+            await expect(page.locator(entryLocator)).toContainText("Science");
+            await expect(page.locator(entryLocator)).toContainText("Driving");
+        }
+
+        // Click Unnamed Clock
+        await page.click('text="Unnamed Clock"');
+
+        // Click Unnamed Notebook
+        await page.click('text="Unnamed Notebook"');
+
+        for (let iteration = 0; iteration < ITERATIONS; iteration++) {
+            const entryLocator = `[aria-label="Notebook Entry"] >> nth = ${iteration}`;
+            await expect(page.locator(entryLocator)).toContainText("Science");
+            await expect(page.locator(entryLocator)).toContainText("Driving");
+        }
+
         //Go to baseURL
         await page.reload();
```

Comment:
There are some timing issues associated with page.reload(). I would try to add some sort of logic to ensure everything has stabilized and is ready to be interacted with.